edxval/api.py

```diff
@@ -104,6 +112,45 @@ def create_video(video_data):
         raise ValCannotCreateError(serializer.errors)
 
 
+def update_video(video_data):
+    """
+    Called on to update Video objects in the database
+
+    update_video is used to update Video objects by the given edx_video_id in the video_data.
+
+    Args:
+        video_data (dict):
+         {
+                url: api url to the video
+                edx_video_id: ID of the video
+                duration: Length of video in seconds
+                client_video_id: client ID of video
+                encoded_video: a list of EncodedVideo dicts
+                    url: url of the video
+                    file_size: size of the video in bytes
+                    profile: ID of the profile
+                courses: Courses associated with this video
+         }
+
```

Comment:
Last nit!

Could you add a return doc? 

`Returns the successfully updated Video object`

and then if you could also add that for the create_video doc as well :)

`Returns the successfully created Video object`